site/release-process.html

```diff
@@ -505,6 +470,34 @@ <h4>Update the Spark Website</h4>
 $ git log v1.1.1 --grep "$expr" --shortstat --oneline | grep -B 1 -e "[3-9][0-9][0-9] insert" -e "[1-9][1-9][1-9][1-9] insert" | grep SPARK &gt; large-patches.txt
 </code></pre>
 
+<h4> Remove RC artifacts from repositories</h4>
+
+<p>After the vote passes and you moved the approved RC to the release repository, you should delete
+the RC directories from the staging repository. For example:</p>
+
+<pre><code>svn rm https://dist.apache.org/repos/dist/dev/spark/v2.3.1-rc1-bin/ \
```

Comment:
the version string in the same should be consistent through out I think? shouldn't this be v2.4.0-rc5?